proposals/0047-content-layer.md

```diff
@@ -304,7 +302,26 @@ const { craft } = Astro.props;
 
 ### Rendered content
 
-Some entry types may have HTML content that can be rendered as a component. While this can be accessed like any other property, a loader can also store the rendered HTML in the `rendered.html` property. This allows users to use the `<Content />` component to render the HTML. The `rendered` property can also include metadata such as frontmatter or headings, which can be accessed as properties on the `rendered.metadata` object.
+Some entry types may have HTML content that can be rendered as a component. While this can be accessed like any other property, a loader can also store the rendered HTML in the `rendered.html` property. This allows users to use the `<Content />` component to render the HTML. The `rendered` property can also include metadata such as frontmatter or headings, which can be accessed as properties on the `rendered.metadata` object:
+
+```ts
+// src/content/config.ts
+store.set({
+  id,
+  data,
+  rendered: {
+    // A raw HTML string
+    html: data.description ?? "",
+    metadata: {
+      // Optionally, metadata such as headings can be stored here
```

Comment:
Can there be anything else then headings?

Reply:
Yes, it's arbitrary, schemaless metadata. Markdown uses it for image paths and raw frontmatter too.